### ISSUE TITLE
Automatic restart of edge module after processing certain chunk of data

### ISSUE THREAD

**Issue by sonal1297** [Azure/iotedge#6232]
## Expected Behavior
The edge module should not restart and should complete with processing the entire dataset that has been provided to it 

## Current Behavior
After processing a chunk of the dataset, the edge module restarts automatically without giving any particular error message. And as a result of this restart, no message is forwarded to other edge modules.

**Comment by huguesBouvier** [comment]
This is very strange, could you provide the logs so we know what's happening?

**Comment by sonal1297** [comment]
@huguesBouvier There are no particular errors that can be seen after running the module or even before the restart of the module. It is just restarting itself after processing a chunk of data.

**Comment by huguesBouvier** [comment]
I'll need more data to tell you what's happening. Right now, I don't know what could be causing the problem.
If you don't have data, please send me a way to repro the issue

**Comment by jlian** [comment]
@sonal1297 can you upload some logs anyway when for when you notice this issue? We can't help troubleshoot without any logs.

**Comment by sonal1297** [comment]
Yes, I am uploading the snippet of logs. Also, can you please @jlian and @huguesBouvier can you give me your mail-ids so that I can send the entire log file via mail
![image](https://user-images.githubusercontent.com/56485128/162434674-499681a7-cf63-47da-ad27-cfef73d858f9.png)

**Comment by huguesBouvier** [comment]
I am sorry I can't put my work email on public channel. This opens up the door for scrapers. What is this edge module? It doesn't look like edgeHub nor edgeAgent?

**Comment by sonal1297** [comment]
@huguesBouvier It is a custom module that has been developed by us.

**Comment by huguesBouvier** [comment]
Could you provide the logs for the following modules:
- EdgeAgent,
- EdgeHub
And for daemon:
journalctl -u aziot-edge

**Comment by varunpuranik** [comment]
@sonal1297 - 
- My guess is that your module is crashing because of some issue, and IoT Edge runtime is restarting it to make it running again. If that is the case, not much we can do. (We don't have context on your custom module, and we won't be able to help you with the error logs from that module)
- If that is not the case, then we should be able to tell from the EdgeAgent debug logs. Can you please enable debug logs for EdgeAgent (set env var RuntimeLogLevel=debug on EdgeAgent with v1.2 or for the entire system like [here](https://docs.microsoft.com/en-us/azure/iot-edge/troubleshoot?view=iotedge-2020-11#check-the-status-of-the-iot-edge-security-manager-and-its-logs)), and provide us EdgeAgent logs for the repro scenario?

**Comment by sarthakvijayvergiya** [comment]
@varunpuranik JFYI we are using Eflow Environment
Could you please share you email address, so we can directly send you the whole support bundle ?

**Comment by varunpuranik** [comment]
@sarthakvijayvergiya - unfortunately I cannot on public forum. Please open a support ticket if you want to do that. 
For this issue, EdgeAgent debug logs should be enough.

**Comment by huguesBouvier** [comment]
Closing issue for inactivity.
